participant/app/routes/$localAgency/recertify/details.tsx

```diff
@@ -45,16 +153,6 @@ export default function Details() {
     relationshipRequired: true,
   };
 
-  const participantCards: ReactElement[] = Array.from({
-    length: participantCount,
-  }).map((it, index) => (
-    <ParticipantCard
-      key={`card-${index}`}
-      index={index + 1}
-      {...participantProps}
-    />
-  ));
-
   return (
     <div>
       <h1>
```

Comment:
I'm not sure why I can't suggest a change here, but can we flip lines 161 and 162 - 63? So the RequiredQuestionStatement goes after the intro text.